src/client/dfs/duns.c

```diff
@@ -708,77 +704,33 @@ duns_create_lustre_path(daos_handle_t poh, const char *path,
 	return rc;
 
 err_cont:
-	daos_cont_destroy(poh, attrp->da_cuuid, 1, NULL);
+	rc2 = daos_cont_destroy(poh, attrp->da_cont, 1, NULL);
+	if (rc2)
+		D_ERROR("Failed to cleanup created container %s (%d)\n", attrp->da_cont, rc2);
```

Comment:
Do we have a standard or enforce formatted errors? In general - beyond this PR - I see different formats/styles. Just wondering if I **should** be leaving comments like this, or if it's not neessary
```suggestion
		D_ERROR("Failed to cleanup created container %s "DF_RC"\n", attrp->da_cont, DP_RC(rc2));
```

Reply:
Definitely agree. That really was a genuine question. I think if we DO want to enforce this in the codebase then maybe it's time for a mass scrub...

Reply:
No we don't have a standard, NLT fault injection checking checks the use of DF_RC but not really much more, although it does at least attempt to ensure that common client error paths are at least not broken.  We also only run it on a tiny proportion of the client codebase so far.